.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "wdi-radio"
+  }
+}
```

Comment:
should upload static files - https://wdidc10.slack.com/archives/discussion/p1468443433001170